<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qualificationinit</class>
 <widget class="QDialog" name="qualificationinit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="PreviousButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SkipButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Skip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NextButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="QuesDownLabel">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>160</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>QuesDownLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="QuesTopLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>QuesTopLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="NameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NameLable</string>
   </property>
  </widget>
  <widget class="QLabel" name="NumberLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NumberLabel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_9">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_10">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
 </widget>
 <tabstops>
  <tabstop>PreviousButton</tabstop>
  <tabstop>SkipButton</tabstop>
  <tabstop>NextButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
